x-pack/plugins/apm/scripts/kibana-security/setup-custom-kibana-user-role.ts

```diff
@@ -147,13 +147,14 @@ async function init() {
 }
 
 async function isSecurityEnabled() {
-  interface XPackInfo {
-    features: { security?: { allow_rbac: boolean } };
+  try {
+    await callKibana({
+      url: `/api/security/v1/me`,
```

Comment:
When you hit that route you get a deprecation message:

```
server    log   [08:02:50.068] [warning][deprecation][plugins][routes][security] The "/kbn/api/security/v1/me" endpoint is deprecated and will be removed in the next major version.
```

and it redirects to /internal/security/me, which is what we should be using.

Reply:
we discussed using `/internal/security/me` but it sounded like a super-private thing 😃 thanks for noticing the redirect

Reply:
Just now noticing that we use `/internal` all over the place 🤦🏻‍♂️